src/app/models/roles.enum.ts

```diff
@@ -3,6 +3,7 @@ export enum Roles {
   TechRecordView = 'CVSFullAccess,TechRecord.View',
   TechRecordCreate = 'CVSFullAccess,TechRecord.Create',
   TechRecordAmend = 'CVSFullAccess,TechRecord.Amend',
+  TechRecordArchive = 'CVSFullAccess,TechRecord.Archive',
```

Comment:
I'm adding a new role for desk-based test creation and found that the roles here don't look right. Shouldn't each role have only one "action"? e.g. `TechRecordArchive = 'TechRecord.Archive',`.

When we log in, roles are stored a string array: 
![image](https://user-images.githubusercontent.com/102959178/203083870-05af5b73-6a99-4105-bf6b-a46e072b7f1f.png)
So then doing something like:
```
userService.roles$.subscribe(roles => {
   if(roles.includes(Roles. TechRecordArchive)) {
      console.log('I can archive');
   }
)
```
This will not log out because there is no role in state with value `CVSFullAccess,TechRecord.Archive`.

Reply:
They're like this so locally you don't need to worry about roles, the production code looks like what you are thinking. There is an env var if you want to toggle it on or off.